pyomo/contrib/mindtpy/initialization.py

```diff
@@ -20,8 +24,9 @@ def MindtPy_initialize_master(solve_data, config):
     """
     m = solve_data.mip = solve_data.working_model.clone()
     MindtPy = m.MindtPy_utils
+    m.dual.deactivate()
 
-    m.dual.activate()
+    # m.dual.activate()
```

Comment:
What is the motivation for deactivating the duals here if you deactivate them in max_binary again and you sort of need them in init_rNLP?

Reply:
Here, m is the master problem which is a MIP. Therefore, we deactivate the dual for MIP. Otherwise, some warnings will be reported.